lib/src/configuration.dart

```diff
@@ -149,7 +149,7 @@ class Configuration {
 
   void _findAndSetCliPath() {
     if (Platform.isMacOS) {
-      _setCliPath("Darwin-x86_64");
+      _setCliPath("Darwin-universal");
```

Comment:
That requires updating https://github.com/getsentry/sentry-dart-plugin/blob/cc136ca78825d28dc08c2a231ea6b7129d1215b0/lib/src/configuration.dart#L150-L162
And with that, we could close https://github.com/getsentry/sentry-dart-plugin/issues/10

Reply:
Ops, missed that, thanks.